src/hooks/useDydxClient.tsx

```diff
@@ -122,10 +123,17 @@ const useDydxClientContext = () => {
     defaultValue: SelectedGasDenom.USDC,
   });
 
+  useEffect(() => {
+    if (compositeClient) {
+      setSelectedGasDenom(gasDenom);
+    }
+  }, [compositeClient]);
```

Comment:
do we also want to add gasDenom to the dep array?

Reply:
would that infinite loop?

Reply:
or I guess this is just for initialization, should be fine then.

Reply:
Yeah. Adding that would cause an infinite loop